model/src/test/java/com/javahelp/model/user/ProviderUserInfoTest.java

```diff
@@ -51,6 +51,19 @@ public void testSetCertified() {
         assertTrue(p.isCertified());
     }
 
+    @Test(timeout = 50)
+    public void testSetGender(){
+        p.setGender("female");
+        assertEquals("FEMALE", p.getGender());
+
+        try{
+            p.setGender("ABC");
+        }
+        catch(IllegalArgumentException e){
+            System.out.println("Invalid input for gender. Please try again");
```

Comment:
When testing for exceptions, instead of using try catches, which do not confirm an exception was actually caught, try using the `expected` field in the `@Test` annotation.

Reply:
Sure let me apply that